src/main/java/dax/walker/utils/AccurateMouse.java

```diff
@@ -126,6 +126,8 @@ public static boolean action(Clickable clickable, boolean hover, String... click
             model = rsObject.getModel();
         } else if (clickable instanceof RSItem) {
             name = RSItemHelper.getItemName((RSItem) clickable);
+        } else if (clickable instanceof RSTile) {
```

Comment:
I wouldn't recommend using this method to click rstiles. What's wrong with walkScreenTile?

Reply:
it wont right click and select walk it will just try and find a small spot that is walkable and click it sometimes will miss-click and loot the ground item, attack mob etc